services/QuillLMS/client/app/bundles/Staff/utils/comprehension/ruleAPIs.ts

```diff
@@ -1,8 +1,16 @@
 import { RuleInterface } from '../../interfaces/comprehensionInterfaces';
 import { handleApiError, apiFetch } from '../../helpers/comprehension';
 
-export const fetchRules = async (key: string, activityId: string) => {
-  const response = await apiFetch(`activities/${activityId}/rules`);
+export const fetchRules = async (key: string, activityId: string, promptId?: any, ruleType?: string) => {
+  let url = `activities/${activityId}/rules`;
```

Comment:
You could make this url calculation its own method to simplify this one.

Reply:
good point, I'll do that for this and `fetchModels`